src/components/backgroundBadge/BackgroundBadge.tsx

```diff
@@ -0,0 +1,20 @@
+import s from "./backgroundBadge.module.scss";
+import IconBg from "../../assets/bg.svg";
+
+interface BackgroundBadgeProps {
+  onClick: () => void;
+  isBgSelect: boolean;
+}
+
+function BackgroundBadge({ isBgSelect, onClick }: BackgroundBadgeProps) {
+  return (
+    <div className={s.container}>
+      <IconBg
```

Comment:
All svg format assemblies to the appropriate folder.
Make a universal component for icons that accepts the icon name parameter - this is the union type of available icon names

Reply:
fixed